ranger/config/commands.py

```diff
@@ -1212,6 +1212,100 @@ def execute(self):
             fm.notify("files have not been retagged")
 
 
+class bulk(Command):
+    """:bulk <attribute>
+
+    This command opens a list of with the attribute <attribute> for each of the
+    selected files in an external editor.
+    After you edit and save the file, it will generate a shell script
+    which changes the attributes in bulk according to the changes you did in the
+    file.
+
+    This shell script is opened in an editor for you to review.
+    After you close it, it will be executed.
+    """
+
+
+    class id3art(object):
+
+        def get_attribute(self, file):
+            import eyed3
+            return str(eyed3.load(file.relative_path).tag.artist)
+
+
+        def get_change_attribute_cmd(self, file, old, new):
+            from ranger.ext.shell_escape import shell_escape as esc
+            return "eyeD3 -a %s %s" % (esc(new), esc(file))
+
+    class id3tit(object):
+
+        def get_attribute(self, file):
+            import eyed3
+            return str(eyed3.load(file.relative_path).tag.title)
+
+
+        def get_change_attribute_cmd(self, file, old, new):
+            from ranger.ext.shell_escape import shell_escape as esc
+            return "eyeD3 -t %s %s" % (esc(new), esc(file))
+
+    bulk = {'id3art': id3art(),
+            'id3tit': id3tit(),
+            }
+
+    def execute(self):  # pylint: disable=too-many-locals,too-many-statements
+        import sys
+        import tempfile
+        from ranger.container.file import File
+        from ranger.ext.shell_escape import shell_escape as esc
+        py3 = sys.version_info[0] >= 3
```

Comment:
Please use the PY3 constant we have defined.

Reply:
Done. This is also done in `bulkrename`, which is where I copied it from. But I only fixed it in `bulk` and not in `bulkrename`, since you're expecting to replace `bulkrename` with `bulk`.